src/lib/Scenes/Artwork/Components/ArtworkHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Flex, Spacer } from "@artsy/palette"
+import { Artwork_artwork } from "__generated__/Artwork_artwork.graphql"
+import React from "react"
+import { ArtworkActionsFragmentContainer as ArtworkActions } from "./ArtworkActions"
+import { ImageCarouselFragmentContainer as ImageCarousel } from "./ImageCarousel/ImageCarousel"
+
+interface ArtworkHeaderProps {
+  artwork: Artwork_artwork
```

Comment:
You should try to avoid using types of a fragment container that is not used in this very module. As the aforementioned comments, simply add a fragment container here that fetched just what you need, which keeps everything very clear.